urbanopt/urbanopt-ditto-reader
# Cli

Resolves #10 

Python CLI to run OpenDSS for URBANopt using the ditto-reader.

To test:

- Pull the branch, do `pip install -e .`
- Do `ditto_reader_cli -h`
- To get help with a command inside the cli, you can do `ditto_reader_cli run-opendss -h`
- For live testing:
    - `Run` and `default process` an urbanopt scenario with electrical features. Ensure you use a feature file that includes a complete electrical network (for instance: `example_project_with_electric_network.json` from the default urbanopt project dir).
    - Call the cli command according to the cli help text.
